app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
+<<<<<<< HEAD
```

Comment:
fix conflict please

Reply:
not fix yet

Reply:
can you help me? I thought it is solved. What should I do to solve?